scripts/stablex/add_token_list.js

```diff
@@ -0,0 +1,21 @@
+const { addTokens } = require("./utilities.js")
+const fetch = require("node-fetch")
+
+module.exports = async function (callback) {
+  try {
+    const token_list = await (await fetch("https://raw.githubusercontent.com/gnosis/dex-react/develop/src/api/tokenList/tokenList.json")).json()
```

Comment:
nit:
I think this file should be provided as a parameter, so it works with any file.

This would allow, to easily approve any lists of tokens in any network. 
For that, I find it ok to commit the token list in this project too. We could leave a NPM script to update it if we want to make it easier to keep them updated. We don't plan to change the list a lot though.